app/pages/profile/comment-link.cjsx

```diff
@@ -57,13 +57,13 @@ module.exports = React.createClass
           <span>
             {' '}in{' '}
             <a href={@state.href}>
-              {if @state.project? and @state.owner
+              {if @state.project? and @state.owner? and !@props.project?
```

Comment:
I also just realized how kludgey and request heavy this component is too, but I'm happy to refactor in a separate PR later.